### DISCUSSION TITLE
multiple Bluetooth trackers

### DISCUSSION THREAD

**Discussion by sudo-joe** [nzymedefense/nzyme#1154] [Questions]
Hello

if a second Bluetooth tracker is required (in addition to the onboard BT-chipset), 
should this be done preferably with a LAN-connected(RJ45) Raspberry Pi with 
installed nzyme-tap_2.0.0-alpha.14_ubuntu2204_amd64.deb?

Any additional config required in nzyme-tap.conf? Just ent0 enabling?

 
Thank's a lot!

**Comment by lennartkoopmann** [comment]
You can just connect an external USB Bluetooth adapter to the same tap computer.
